src/views/UserManagement.vue

```diff
@@ -296,17 +293,10 @@ export default defineComponent({
           title: 'Access Level',
           fieldNames: ['acl'],
           displayFunction: (item: User) => {
-            if (item.acl === '1') {
-              return 'Viewer'
-            } else if (item.acl === '11') {
-              return 'Editor'
-            } else if (item.acl === '20') {
-              return 'Organization Admin'
-            } else if (item.acl === '100') {
-              return 'Reblaze Admin'
-            } else {
-              return undefined
-            }
+            const accessLevel = accessLevels.find((accessLevel) => {
+              return accessLevel.value === item.acl
+            })
+            return accessLevel.title
```

Comment:
What if the accessLevel was not found?

```suggestion
            return accessLevel?.title || ''
```

Reply:
Right, added